tardis/tests/test_tardis_full_formal_integral.py

```diff
@@ -89,11 +90,11 @@ def test_spectrum(self, simulation, request):
         luminosity = simulation.spectrum_solver.spectrum_real_packets.luminosity
         expected = regression_data.sync_ndarray(luminosity.cgs.value)
         expected = u.Quantity(expected, "erg /s")
-        assert_quantity_allclose(luminosity, expected)
+        assert_quantity_allclose(luminosity, expected, rtol=1e-11, atol=0*u.erg/u.s)
```

Comment:
Why is this needed? Sorry if this has been asked before

Reply:
for all the tests we do - we always want to make sure to know exactly the tolerance.